demos-europe/demosplan-core
# feat(refs T31974): Add pager for statements and segments

**Ticket:** https://yaits.demos-deutschland.de/T31974



This PR adds the feature that pages which were selected in the statement list's sliding pagination will be persisted in the session storage, even if the user is leaving the current view.

### How to review/test

navigate to the statement list, select a sliding pagination page and leave the site. The last selected page should be mounted when re-visiting the view.

### Linked PRs (optional)
(https://github.com/demos-europe/demosplan-ui/pull/175)

### Tasks (optional)


### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
